### DISCUSSION TITLE
Good Job guys

### DISCUSSION THREAD

**Discussion by jonathanpmartins** [quasarframework/quasar#9019] [Ideas / Proposals]
I just wanna congratulate everyone for the good job making the upgrade material. At first I got stuck with some minor issues, but they were solved pretty fast!
I'm very happy that I choose the right framework to develop my last endeavors!
Cheers

**Comment by rstoenescu** [comment]
Thank you for the nice words!

We're interested in the minor issues as well if you can specify them. A lot of effort has been put into the Qv2 upgrade guide to be as smooth and complete as possible, but as always we are open to feedback into making it even better.
